remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,22 @@
+import remotespark.utils.configuration as conf
+import importlib
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        events_handler = importlib.import_module(conf.events_handler_class())
+        self.handler = events_handler()
+
+    def emit_start_session_event(self, time_stamp, event_name, session_guid, language):
```

Comment:
What's the dimensions for each of these events?

I believe you don't need to pass in the `event_name` or `time_stamp`. Both of these fields can be handled by this class.